src/css/map-view.css

```diff
@@ -1472,34 +1473,109 @@ other class: .ui-slider-range */
 .viewfinder-zoom-presets {
   display: flex;
   flex-direction: column;
+  margin-top: 1rem;
+  padding: .25rem .5rem;
   row-gap: .25rem;
+  color: black;
 
-  .viewfinder-zoom-presets__preset {
+  .viewfinder-zoom-preset__preset {
     border-radius: var(--map-border-radius-small);
     border: 1px solid var(--map-col-section-divider, var(--map-col-bkg-lightest__deprecate));
     cursor: pointer;
-    padding: .5rem;
+    display: flex;
+    flex-direction: column;
+    padding: .5rem 2rem;
+    row-gap: .5rem;
 
     &:hover {
```

Comment:
Since it's a clickable element, I'm using the same hover state as expansion panel and place predictions. I can confirm with Doug, but typically I think it's safe to extrapolate on the mocks

Reply:
Apparently the mocks have components that we can drill into, from Doug: https://www.figma.com/file/9R80DBDAMQtOsu8y1HkAVT/permafrost-discovery-gateway?type=design&node-id=23-6574&mode=design&t=t6Xv4PMAR4UNntsY-0

Some additional color changes on layer labels as well

Reply:
cool! (but also gaaasp one more set of colors to name!)